IMAGE-LICENSE

```diff
@@ -0,0 +1,15 @@
+Image-Licenses
+==============
+
+Here you can find the license information about the images used.
+
+Sprite-Images:
+--------------
+
+| Sprite Name | Original Name  | Source-URL |
+| ------- | --- | --- |
```

Comment:
The actual licensing information is missing :-)

The license for these images is this one: https://creativecommons.org/licenses/by/3.0/us/legalcode
Could you add another section "License Texts" with a sub-section "CC BY 3.0 US" that contains this link (at the end of the file)? I'd like to paste the complete license text, but unfortunately it isn't available in plain text, so the link has to be enough for now.

Could you also add two further columns to the table?
- One should contain the creator of each image, as given on the respective thenounproject.com pages.
- One should contain the license. All of these images use "CC BY 3.0 US"

Reply:
done